src/store/mode/actions.ts

```diff
@@ -0,0 +1,6 @@
+import { createAction } from '@reduxjs/toolkit';
+import { TOGGLE_ANNOTATION_MODE, Mode } from './types';
+
+const toggleAnnotationModeAction = createAction<Mode>(TOGGLE_ANNOTATION_MODE);
+
+export default toggleAnnotationModeAction;
```

Comment:
Should this export an object to allow for more than one selector? Same for the actions.